packages/vitest-file-snapshots/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@cronn/vitest-file-snapshots",
+  "version": "0.4.1",
+  "description": "Write tests with Vitest using file snapshots",
+  "keywords": ["vitest", "file snapshots"],
+  "bugs": {
+    "url": "https://github.com/cronn/vitest-file-snapshots/issues"
+  },
+  "author": "cronn",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cronn/vitest-file-snapshots.git",
+    "directory": "packages/vitest-file-snapshots"
+  },
+  "homepage": "https://github.com/cronn/vitest-file-snapshots",
+  "engines": {
+    "node": ">=22",
+    "pnpm": ">=10"
+  },
+  "scripts": {
+    "check": "biome check .",
+    "check:fix": "biome check --write .",
+    "test": "vitest run",
+    "test:coverage": "vitest run --coverage",
+    "compile": "tsc",
+    "build": "tsup && node scripts/fix-declaration-file.js",
+    "ci": "biome ci . && pnpm run compile & pnpm run test:coverage && pnpm run build"
+  },
+  "type": "module",
+  "exports": {
+    "./package.json": "./package.json",
+    "./matcher": {
+      "types": "./dist/matcher.d.ts",
+      "default": "./dist/matcher.js"
+    }
+  },
+  "files": ["dist"],
+  "dependencies": {
+    "@cronn/lib-file-snapshots": "workspace:*"
+  },
```

Comment:
How will the dependency on `@cronn/lib-file-snapshots` work? As far as I can see each package will have it's own version number(?). Will there be a dependency on an exact version of `lib-file-snapshots`? Or on a `^` range?